WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -448,15 +459,17 @@ private extension CollectOrderPaymentUseCase {
 // MARK: MailComposer Delegate
 extension CollectOrderPaymentUseCase: MFMailComposeViewControllerDelegate {
     func mailComposeController(_ controller: MFMailComposeViewController, didFinishWith result: MFMailComposeResult, error: Error?) {
+        let cardReaderModel = connectedReader?.readerType.model ?? ""
```

Comment:
As above for "should we track an empty string or not?"

Reply:
(answered above)